ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -41,9 +41,8 @@
   private final AtomicBoolean open = new AtomicBoolean(true);
   private final Logger logger = LoggerFactory.getLogger(getClass());
 
-  private static final short VERSION = 0;
+  private static final short VERSION = 1;
```

Comment:
perhaps you could have `VERSION_0` and `VERSION_1` defined for this class as well.